File: backend/mailbook/praktika/models/location.py
from django.db import models


class Location(models.Model):
    region = models.OneToOneField("Region",verbose_name="Регион",on_delete=models.CASCADE)
    city = models.CharField(max_length=255,verbose_name="Город")
    microdistrict = models.CharField(max_length=255,verbose_name="Микрорайон")
    house_num = models.IntegerField(verbose_name="Номер дома")
    apartment = models.IntegerField(verbose_name="Номер квартиры")


    def __str__(self):
        return f'{self.city} {self.microdistrict}'

File: backend/src/models/enteties/location.ts
import {
    BaseEntity, Column,
    Entity, OneToMany,

    PrimaryGeneratedColumn
} from "typeorm"



@Entity('location')
export class Location extends BaseEntity {
    @PrimaryGeneratedColumn("uuid")
    id: string;

    @Column()
    region:string

    @Column()
    city:string

    @Column()
    microdistrict:string

    @Column()
    house_num:number

    @Column()
    apartment:number

}